app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -17,14 +19,13 @@ define([
       return account.getAvatar()
         .then(function (result) {
           if (result && result.avatar && result.id) {
-            self.logEvent(self.className + '.profile_image_shown');
-          } else {
-            self.logEvent(self.className + '.profile_image_not_shown');
+            return ImageLoader.load(result.avatar);
```

Comment:
It seems like this section (lines 22-27) should be in their own model. For another PR.